### PR TITLE
2.x: Add a sentence to documentation of take() operator.

### PR DESCRIPTION
**Pull request by sadegh**
#5716

### CODE REVIEW

File: src/main/java/io/reactivex/Flowable.java [hunk 1]
```diff
@@ -13391,6 +13391,8 @@ public final void subscribe(FlowableSubscriber<? super T> s) {
     /**
      * Returns a Flowable that emits those items emitted by source Publisher before a specified time runs
      * out.
+     * If time runs out before Flowable is finished, termination events would be balled on provided {@link Scheduler},
```
Comment (akarnokd) [inline src/main/java/io/reactivex/Flowable.java]: A `<p>` would cleanly separate the first sentence and the rest of the Javadoc. Also the wording could use some refinement:

```
<p>
If time runs out before the {@code Flowable} completes normally, the {@code onComplete} 
event will be signaled on the default {@code computation} {@link Scheduler}.
```

File: src/main/java/io/reactivex/Flowable.java [hunk 2]
```diff
@@ -13418,6 +13420,8 @@ public final void subscribe(FlowableSubscriber<? super T> s) {
     /**
      * Returns a Flowable that emits those items emitted by source Publisher before a specified time (on a
      * specified Scheduler) runs out.
+     * If time runs out before Flowable is finished, termination events would be balled on provided {@link Scheduler},
```
Comment (akarnokd) [inline src/main/java/io/reactivex/Flowable.java]:
```
<p>
If time runs out before the {@code Flowable} completes normally, the {@code onComplete} 
event will be signaled on the provided {@link Scheduler}.
```

File: src/main/java/io/reactivex/Observable.java [hunk 3]
```diff
@@ -11285,6 +11285,8 @@ public final void subscribe(Observer<? super T> observer) {
     /**
      * Returns an Observable that emits those items emitted by source ObservableSource before a specified time runs
      * out.
+     * If time runs out before Observable is finished, termination events would be called on provided {@link Scheduler},
```
Comment (akarnokd) [inline src/main/java/io/reactivex/Observable.java]:
```
<p>
If time runs out before the {@code Observable} completes normally, the {@code onComplete} 
event will be signaled on the default {@code computation} {@link Scheduler}.
```

File: src/main/java/io/reactivex/Observable.java [hunk 4]
```diff
@@ -11308,6 +11310,8 @@ public final void subscribe(Observer<? super T> observer) {
     /**
      * Returns an Observable that emits those items emitted by source ObservableSource before a specified time (on a
      * specified Scheduler) runs out.
+     * If time runs out before Observable is finished, termination events would be called on provided {@link Scheduler},
```
Comment (akarnokd) [inline src/main/java/io/reactivex/Observable.java]:
```
<p>
If time runs out before the {@code Flowable} completes normally, the {@code onComplete} 
event will be signaled on the provided {@link Scheduler}.
```

**Review by akarnokd — CHANGES_REQUESTED** [review]

**Review by vanniktech — COMMENTED** [review]
Agreeing with david

**Review by vanniktech — APPROVED** [review]

**Review by akarnokd — APPROVED** [review]

**Pull request merged**